vwifi.c

```diff
@@ -150,6 +154,81 @@ MODULE_PARM_DESC(station, "Number of virtual interfaces running in STA mode.");
 /* Global context */
 static struct owl_context *owl = NULL;
 
+/* Blocklist content */
+#define MAX_NETLINK_USER 31
```

Comment:
Move the definitions out of C source. They should appear in a common header shared by Linux kernel module and our tool.

Reply:
I've remove this macro and use the definition in linux kernel, which is `MAX_LINKS`.
Reference : https://android.googlesource.com/platform/bionic/+/a799b53f10e5a6fd51fef4436cfb7ec99836a516/libc/kernel/common/linux/netlink.h#36